recipes/ott-jax/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "ott-jax" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ott-jax-{{ version }}.tar.gz
+  sha256: adcd3c59ac9223a962b7a9244c4f53cc9857bf2461530583edb1ea8a1fc37ea3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<37]
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - jax >=0.1.67
+    - jaxlib >=0.1.47
+    - numpy >=1.18.4,!=1.23.0
+    - matplotlib-base >=3.0.0
+    - flax >=0.5.2
+    - optax >=0.1.1
+    - scipy >=1.7.0
+    - importlib-metadata >=1.0  # [py<38]
+    - typing-extensions  # [py<38]
```

It is fine to drop these selectors here for the benefit of keeping it noarch.

```suggestion
    - importlib-metadata >=1.0
    - typing-extensions
```

---

Agree, but even with `noarch: python`, it was the CI for Windows failed.